src/pages/gradient-text.ts

```diff
@@ -40,6 +41,7 @@ const getRemoveColorButtonElement = getRemoveNewColorButton(attribute);
 
 const getDegreeElement = getRange(attribute);
 const resetButton = getResetButton(attribute);
+const getTailwindCode = getTailwindButton(attribute);
 
 function copyHandler() {
```

Comment:
we also already have a copy handler, once again you can just duplicate for the tailwind, makes this easier to understand